coordinator/src/main/java/com/tencent/rss/coordinator/CoordinatorFactory.java

```diff
@@ -18,23 +18,26 @@
 
 package com.tencent.rss.coordinator;
 
+import com.tencent.rss.common.metrics.GRPCMetrics;
 import com.tencent.rss.common.rpc.GrpcServer;
 import com.tencent.rss.common.rpc.ServerInterface;
 
-public class CoordinatorRpcServerFactory {
+public class CoordinatorFactory {
 
   private final CoordinatorServer coordinatorServer;
   private final CoordinatorConf conf;
+  private final GRPCMetrics grpcMetrics;
 
-  public CoordinatorRpcServerFactory(CoordinatorServer coordinatorServer) {
+  public CoordinatorFactory(CoordinatorServer coordinatorServer, GRPCMetrics grpcMetrics) {
```

Comment:
Do we need input param grpcMetrics since it is a member of coordinatorServer?

Reply:
ok